rpcs3/Emu/System.h

```diff
@@ -261,7 +261,7 @@ struct cfg_root : cfg::node
 	{
 		node_core(cfg::node* _this) : cfg::node(_this, "Core") {}
 
-		cfg::_enum<ppu_decoder_type> ppu_decoder{this, "PPU Decoder", ppu_decoder_type::fast};
+		cfg::_enum<ppu_decoder_type> ppu_decoder{this, "PPU Decoder", ppu_decoder_type::llvm};
```

Comment:
On Linux, I think you can compile RPCS3 without the LLVM recompiler. Do you know what would happen in this case with this change ?